documentation/modules/ROOT/pages/connectors/spanner.adoc

```diff
@@ -0,0 +1,1201 @@
+|===
+|Attributes |Type |Description
+
+|[[connectors-strm-metric-millisecondssincelastevent]]<<connectors-strm-metric-millisecondssincelastevent, `+MilliSecondsSinceLastEvent+`>>
+|`long`
+|The number of milliseconds since the connector has read and processed the most recent event.
+
+|[[connectors-strm-metric-totalnumberofeventsseen]]<<connectors-strm-metric-totalnumberofeventsseen, `+TotalNumberOfEventsSeen+`>>
+|`long`
+|The total number of events that this connector has seen since last started or reset.
+
+|[[connectors-strm-metric-numberofeventsfiltered]]<<connectors-strm-metric-numberofeventsfiltered, `+NumberOfEventsFiltered+`>>
+|`long`
+|The number of events that have been filtered by include/exclude list filtering rules configured on the connector.
+
+|[[connectors-strm-metric-queuetotalcapacity]]<<connectors-strm-metric-queuetotalcapacity, `+QueueTotalCapacity+`>>
+|`int`
+|The length the queue used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-queueremainingcapacity]]<<connectors-strm-metric-queueremainingcapacity, `+QueueRemainingCapacity+`>>
+|`int`
+|The free capacity of the queue used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-connected]]<<connectors-strm-metric-connected, `+Connected+`>>
+|`boolean`
+|Flag that denotes whether the connector is currently connected to the database server.
+
+|[[connectors-strm-metric-millisecondsbehindsource]]<<connectors-strm-metric-millisecondsbehindsource, `+MilliSecondsBehindSource+`>>
+|`long`
+|The number of milliseconds between the last change event's timestamp and the connector processing it.
+The values will incoporate any differences between the clocks on the machines where the database server and the connector are running.
+
+|[[connectors-strm-metric-numberofcommittedtransactions]]<<connectors-strm-metric-numberofcommittedtransactions, `+NumberOfCommittedTransactions+`>>
+|`long`
+|The number of processed transactions that were committed.
+
+|[[connectors-strm-metric-maxqueuesizeinbytes]]<<connectors-strm-metric-maxqueuesizeinbytes, `+MaxQueueSizeInBytes+`>>
+|`long`
+|The maximum buffer of the queue in bytes used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-currentqueuesizeinbytes]]<<connectors-strm-metric-currentqueuesizeinbytes, `+CurrentQueueSizeInBytes+`>>
+|`long`
+|The current buffer of the queue in bytes used to pass events between the streamer and the main Kafka Connect loop.
+
+|[[connectors-strm-metric-lowwatermark]]<<connectors-strm-metric-lowwatermark, `+LowWatermark+`>>
+|`long`
+|The current low watermark of the connector task. The low watermark describes the time T at which the connector is guaranteed to have streamed out all events with timestamp < T.
+
+|[[connectors-strm-metric-latencylowwatermarklag]]<<connectors-strm-metric-latencylowwatermarklag, `+LatencyLowWatermarkLag+`>>
```

Comment:
Please corect the metric name, should be like `latencyLowWatermarkLag<variant>MilliSeconds`. Please not the basic name was not correct and I introduced a mask-format where `<variant>` will be the P50 etc.
Same issue is for the following metics.

Reply:
Sure, to be clear, are you asking me to rename it to: latencyLowWatermarkLag<variant>MilliSeconds?

And I will specify in the comments that variant can be P50, P95, P99, etc.